app/models/location.py

```diff
@@ -72,3 +72,21 @@ class Address(db.Model):
 
     def __repr__(self):
         return '<Address \'%s\'>' % self.name
+
+    @staticmethod
+    def generate_fake(count=10):
+        """Generate count fake Resources for testing."""
+        from faker import Faker
+
+        fake = Faker()
+        from random import choice
+        for i in range(count):
+            a = Address(
+                name=fake.name(),
+                street_address=fake.street_address(),
+                city=fake.city(),
+                state=fake.state(),
+                zip_code=choice(ZIPCode.query.all())
```

Comment:
we probably want to store this as a variable (perhaps named `zip_codes`) then we can say `zip_code=choice(zip_codes)` so we don't have to query each time.